src/Components/WebAssembly/WebAssembly/src/HotReload/WebAssemblyHotReload.cs

```diff
@@ -24,39 +19,11 @@ namespace Microsoft.AspNetCore.Components.WebAssembly.HotReload
     [EditorBrowsable(EditorBrowsableState.Never)]
     public static class WebAssemblyHotReload
     {
-        private static readonly ConcurrentDictionary<Guid, List<(byte[] metadataDelta, byte[] ilDelta)>> _deltas = new();
-        private static readonly ConcurrentDictionary<Assembly, Assembly> _appliedAssemblies = new();
-        private static (List<Action<Type[]?>> BeforeUpdates, List<Action<Type[]?>> AfterUpdates)? _handlerActions;
-
-        static WebAssemblyHotReload()
+        private static HotReloadAgent? _hotReloadAgent;
+        private static UpdateDelta[] _updateDeltas = new[]
```

Comment:
Why is this a list if we're ever only managing a single delta at a time?

Reply:
I wanted to keep the HotReloadAgent code identical. Right now, the blazor webassembly hot reload, sends deltas one at a time for "reasons" so using an array works. It would be worthwhile to switch it up and have it receive all of the deltas simultaneously, but that would be a little more effort to plumb thru. Another day!